docs/learning/WriteAConnector.md

```diff
@@ -585,6 +589,98 @@ By following the job-subject scoping rule, many connectors can write data to a s
 
 Job-subject scoping also prevents problems with locks and codes. The codes used by a Connector are scoped by the job subject, and there should be no risk of conflicts and hence codes are not reserved.
 
+### Unmap
+
+Sometimes it becomes necessary to 'undo' a connector run and an extreme step of deleting an imodel and starting over is not practical because, for example, the imodel contains the content of multiple connectors and a more surgical approach is warranted.  **shouldUnmapSource** is a job argument (i.e. this.jobArgs.shouldUnmapSource) that can be passed to the ConnectorRunner and it will call the connector's **unmapSource** method.
```

Comment:
I don't think unmapping is really an edge condition. Data may have just moved from one file to another... maybe a file was added to a list by mistake... maybe there's a need to update the document's ID...
I think stressing that iModels are multi-source still makes sense, but unmapping shouldn't be described as something particularly special, so let's try to soften that.

> The Unmap mode allows removing all data from a given input source from the iModel. This is important because iModels contains data from many sources, and users may need to selectively remove data (e.g. maybe it moved, maybe there's now a better Connector, or maybe it was added by mistake). **shouldUnmapSource** is a job argument (i.e. this.jobArgs.shouldUnmapSource) that can be passed to the ConnectorRunner and it will call the connector's **unmapSource** method.

Reply:
@jffmarker thanks for the suggestion. just pushed update.